kotlin-native/NEW_MM.md

```diff
@@ -1,44 +1,36 @@
-  `producer` to return an isolated object subgraph.
-
-Also, reference cycles containing `AtomicReference` and `FreezableAtomicReference` do not cause memory leaks anymore.
-
-A few caveats:
-* As with the previous MM, memory is not reclaimed eagerly: an object is reclaimed only when GC happens. This extends
-  to Swift/ObjC objects that crossed interop boundary into Kotlin/Native.
-* `AtomicReference` from `kotlin.native.concurrent` still requires freezing the `value`. `FreezableAtomicReference`
-  can be used instead, or, alternatively, `AtomicRef` from `atomicfu` can be used (**NOTE**: _`atomicfu` has not reached 1.x yet_).
-* `deinit` on Swift/ObjC objects (and the objects referred by them) will be called on a different thread if these objects
-  cross interop boundary into Kotlin/Native.
+* Top-level properties can be accessed and modified by any thread without using `@SharedImmutable`.
+* Objects passing through interop can be accessed and modified by any thread without freezing them.
+* `Worker.executeAfter` no longer requires `operation` to be frozen.
+* `Worker.execute` no longer requires `producer` to return an isolated object subgraph.
+* Reference cycles containing `AtomicReference` and `FreezableAtomicReference` do not cause memory leaks anymore.
+
+A few precautions:
+* As with the previous MM, memory is not reclaimed eagerly: an object is reclaimed only when GC happens. This extends to Swift/ObjC objects that crossed interop boundary into Kotlin/Native.
+* `AtomicReference` from `kotlin.native.concurrent` still requires freezing the `value`. Instead, you can use `FreezableAtomicReference` or `AtomicRef` from `atomicfu`. _Note that `atomicfu` has not reached 1.x yet_.
+* `deinit` on Swift/ObjC objects (and the objects they refer to) will be called on a different thread if these objects cross interop boundary into Kotlin/Native.
 * When calling Kotlin suspend functions from Swift, completion handlers might be called on threads other than the main.
 
-Together with the new MM we are bringing in another set of changes:
-* Global properties are initialized lazily, when the file they are defined in is first accessed. Previously global properties were
-  initialized at the program startup.
-  This is in line with Kotlin/JVM. As a workaround, properties that must be initialized at the program start can be marked with `@EagerInitialization`
-  (please, consult the docs for `@EagerInitialization` before using).
-* `by lazy {}` properties support thread safety modes and do not handle unbounded recursion. This is in line with Kotlin/JVM.
-* Exceptions escaping `operation` in `Worker.executeAfter` are processed like in other parts of the runtime:
-  by trying to execute a user-defined unhandled exception hook, or terminating the program if the hook was not found or
-  failed with exception itself.
+Together with the new MM, we're bringing in another set of changes:
```

Comment:
`The new MM also brings another set of changes` sounds more natural to me, but this version is also acceptable.